framework/encode/vulkan_state_tracker.cpp

```diff
@@ -1935,6 +1935,7 @@ void VulkanStateTracker::DestroyState(vulkan_wrappers::SwapchainKHRWrapper* wrap
     std::unique_lock<std::mutex> lock(state_table_mutex_);
     for (auto entry : wrapper->child_images)
     {
+        DestroyState(entry);
```

Comment:
Does this need similar logic to https://github.com/LunarG/gfxreconstruct/blob/dev/framework/encode/vulkan_handle_wrapper_util.h#L650-L658 to handle child images that belong to multiple swapchains?

Reply:
Good catch

Reply:
Maybe not, the most recent push is now crashing on replay